src/main/java/uk/gov/register/presentation/config/Field.java

```diff
@@ -32,7 +34,7 @@ public Field(@JsonProperty("field") String fieldName,
     }
 
     public Cardinality getCardinality() {
-        return cardinality;
+        return Stream.of(Cardinality.values()).filter(c -> Objects.equals(c.getId(), cardinality)).findFirst().get();
```

Comment:
Fail fast is good but in this particular case I am not very keen to do it because we are still in development mode and there are cases where we add some data entry which can not be deserialized because of which the register is down. 

I guess first we should make sure that the data is clean and validated.

Also dropwizard/dropwizard#699 is still an open issue, the changes above are good to simplify the cardinality enum code.